src/scythe-bidder/create-room.tsx

```diff
@@ -59,6 +78,29 @@ export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
                   ))}
               </Select>
             </Form.Item>
+            <Form.Item>
+              <Select<string>
+                value={gameType}
+                onChange={(value) => {
+                  setGameType(value);
+                  if (value === SCYTHE_BASE){
+                    if (numPlayers > MAX_PLAYERS_BASE){
+                      setNumPlayers(MAX_PLAYERS_BASE);
+                      notification.warning({
+                        message     : 'Warning',
+                        description : 'Max players changed',
+                        placement   : 'topLeft',
```

Comment:
This is just personal preference, but I'm personally used to it being in the top right or either bottom corner - basically anything but top left. I think if we do change its position from the default top right, we should change all other notifications used (probably better to set globally with `notification.config`)?

Reply:
Also, we could make the description a little better, like "Only up to 5 players allowed in the base game" or "Only up to X players possible with the combinations you have selected".

Reply:
On my laptop, I almost missed the default notification on the top right because the notification text is short and I'm used to ignoring such pop-ups on other websites. It's better with the updated message, so I changed it back to the default placement.